magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -200,7 +200,7 @@ event_classifier:
 create_irf:
     quality_cuts: "disp_diff_mean < 0.22"
     event_type: "software"  # select "software", "software_only_3tel", "magic_only" or "hardware"
-    weight_type_dl2: "intensity"  # select "simple", "variance" or "intensity"
```

Comment:
I think this change was not intended (the intensity weight is our standard since it provides a bit better performance)

Reply:
Hi, This is the config file used by the tests, and I used 'simple' because, this way, I can check if the results are the expected ones (I can evaluate simple averages by hand and check if the values returned by the 'load' functions are the right ones). Obviously, tests only check the algorithms; the results can't be used to get to some scientific result, so the performance is not really important (anyway, with one LST sub-run and one MC run, the spectrum and the IRFs are meaningless). Before merging, we will have to discuss about where to put this config (with data instead of in the repo? or in the repo but with a different name?)

Reply:
hmmm, something is wrong here. 
The actual config file for tests you also download together with the data (in config subdirectory), so you do not need to include it in the repository. Since config files are small we could do it if needed, but the way how you did it, it overwrites the default config, and this is bad idea, both because this config file is used by the users, and also the default config will be also modified in the future following improvements of the analysis, and then you would need to redo all the tests with the new config

Reply:
Yes I can remove this from the PR (not to overwrite it)

Reply:
Now the configuration files are in the repository, but they are (should be) used as they are, without overwriting any exiting configuration file.